APCAppCore/APCAppCore/Library/Objects/APCKeychainStore.m

```diff
@@ -191,7 +189,6 @@ + (BOOL) removeItemForKey: (NSString *) key
         NSMutableDictionary *itemToDelete = [[NSMutableDictionary alloc] init];
         [itemToDelete setObject:(__bridge id)kSecClassGenericPassword forKey:(__bridge id)kSecClass];
         [itemToDelete setObject:service forKey:(__bridge id)kSecAttrService];
-        [itemToDelete setObject:key forKey:(__bridge id)kSecAttrGeneric];
         [itemToDelete setObject:key forKey:(__bridge id)kSecAttrAccount];
 #if !TARGET_IPHONE_SIMULATOR && defined(__IPHONE_OS_VERSION_MIN_REQUIRED)
```

Comment:
The objects set on the dictionary/query are not consistent between methods, and removeAllItemsForService uses the dictionary in a loop. The only consistent line is ```[query setObject:(__bridge id)kSecClassGenericPassword forKey:(__bridge id)kSecClass];